src/features/import-maps.js

```diff
@@ -33,7 +33,22 @@ function processScripts () {
       script.sp = true;
       if (!script.src)
         return;
-      System.import(script.src.slice(0, 7) === 'import:' ? script.src.slice(7) : resolveUrl(script.src, baseUrl));
+      System.import(script.src.slice(0, 7) === 'import:' ? script.src.slice(7) : resolveUrl(script.src, baseUrl)).catch(function (e) {
+        // if there is a script load error, dispatch an "error" event
+        // on the script tag.
+        if (e.message.indexOf('https://git.io/JvFET#3') > -1) {
+          var event
+          // IE 11 throw when callin new Event()
+          try {
+            event = new Event('error')
+          }
+          catch(_) {
+            event = document.createEvent('error')
```

Comment:
When I try this in IE11 it gives an error that the event type is not supported. It seems this shoudl either be `event` or `errorevent`. Testing in Chrome gives the same results as IE11 though so this isn't hard to check.

Further the following still throws for me in Chrome:

```js
script.dispatchEvent(document.createEvent('error'))
```

with the error that the event is uninitialized.

Both these cases need to be handled it seems

Reply:
Thanks for your help on this. I have updated the code in d4c0fecdd3436613a4dd2cca877ac0bc1eedc059 to

```js
var event = document.createEvent('Event');
event.initEvent('error', false, false);
script.dispatchEvent(event);
```

I have tested it on recent Chrome/Firefox and it works + there is no warning in the console.

Reply:
Just tested this in IE11 and I can confirm it works perfectly there too.